redis/files/redis-2.6.conf.jinja

```diff
@@ -1,45 +1,40 @@
 # Redis config
-{% set redis  = pillar.get('redis', {}) -%}
+{% from "redis/map.jinja" import redis_settings with context %}
 daemonize yes
 
-pidfile {{ redis.get('pidfile', '/var/run/redis.pid') }}
-port {{ redis.get('port', 6379) }}
+pidfile {{ redis_settings.pidfile }}
+port {{ redis_settings.port }}
 
-{% if redis['bind'] is defined %}
-bind {{ redis['bind'] }}
+{% if redis_settings.bind is defined %}
+bind {{ redis_settings.bind }}
 {% endif -%}
 
-{% if redis['unixsocket'] is defined %}
-unixsocket {{ redis['unixsocket'] }}
-unixsocketperm {{ redis.get('unixsocketperm', 755) }}
+{% if redis_settings.unixsocket is defined %}
+unixsocket {{ redis_settings.unixsocket }}
+unixsocketperm {{ redis_settings.unixsocketperm|default(755) }}
 {% endif -%}
 
-tcp-keepalive {{ redis.get('tcp-keepalive', 0) }}
+tcp-keepalive {{ redis_settings.tcp_keepalive }}
 
-loglevel {{ redis.get('loglevel', 'notice') }}
-{% if redis['syslog'] is defined %}
+loglevel {{ redis_settings.loglevel }}
+{% if redis_settings.syslog is defined %}
 syslog-enabled yes
-syslog-ident {{ redis['syslog']['ident'] }}
-syslog-facility {{ redis['syslog']['facility'] }}
-{% if redis['syslog']['nologfile'] is defined %}
+syslog-ident {{ redis_settings.syslog.ident }}
+syslog-facility {{ redis_settings.syslog.facility }}
+{% if redis_settings.syslog.nologfile is defined %}
 logfile ""
 {% else %}
-logfile {{ redis.get('logfile', '/var/log/redis/redis.log') }}
+logfile {{ redis_settings.logfile }}
 {% endif -%}
 {% else %}
-logfile {{ redis.get('logfile', '/var/log/redis/redis.log') }}
+logfile {{ redis_settings.logfile }}
 {% endif -%}
 
-{% if redis['syslog'] is defined %}
-syslog-enabled yes
-syslog-ident {{ redis['syslog']['ident'] }}
-syslog-facility {{ redis['syslog']['facility'] }}
-{% endif -%}
```

Comment:
I think you don't need a replacement, because it's also defined  in line 20-31.
So the old version of this template defined it twice.
Sorry I should have mentioned this in an own commit.